src/runtime-catalogue/RuntimeCatalogue.js

```diff
@@ -373,7 +386,6 @@ class RuntimeCatalogue {
         // check encoding
         if (sp["encoding"] === "base64") {
             sp["sourceCode"] = this.atob(sp["sourceCode"]);
-            sp["encoding"] = 'utf-8';
```

Comment:
i think, this is needed to.. because could generate an error because, the `sp['encoding']` always be the `base64` and it will try to decode again and will throw an error;